scripts/commit-build.sh

```diff
@@ -53,8 +54,8 @@ git config --global user.name "Blackbaud Stache Build User"
 commit_build() {
   echo "Committing build results to ${1}..."
   git status --short
+  git add --all
```

Comment:
Makes sense to move this here. I'm guessing the diff checks weren't working. I don't think the diff includes non-added files?